hack/examples/golang/image/function.yaml

```diff
@@ -23,3 +23,4 @@ spec:
   handler: main:Handler
   minReplicas: 1
   maxReplicas: 1
+  disableDefaultHttpTrigger: false
```

Comment:
why does it needs to be explicitly stated? I think it should be the default

Reply:
@liranbg because if platform's configuration is set to `true`, those examples won't work because they do not contain triggers

Reply:
but why do we need to be explicit about it? 
I think it is better to add this configuration to function configuration readme file and not on each example

Reply:
will add explanations :)